examples/ITK_Example18_MONAI_affine_Elastix_nonlinear.ipynb

```diff
@@ -0,0 +1,577 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #25.        max_epochs=3
</pre><p>Maybe add a comment, saying that the number of epochs is limited for CPU runs (and thereby suboptimal), because otherwise running the example would take too much time.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/InsightSoftwareConsortium/ITKElastix/pull/194/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>